examples/production-deployment-k8s-helm/templates/configmap.yaml

```diff
@@ -4,7 +4,10 @@ metadata:
   name: {{ include "tensorzero.fullname" . }}
   labels:
     {{- include "tensorzero.labels" . | nindent 4 }}
```

Comment:
Happy to just extend `tensorzero.labels` in `_helpers.tpl` if we prefer that approach. Could be less annoying to maintain over time.

Reply:
That might be less error-prone in case we add more resources in the future, let's do that, thanks!

Reply:
Done